k8s/namespaces/divorce/div-dgs/div-dgs.yaml

```diff
@@ -15,3 +15,4 @@ spec:
       image: hmctspublic.azurecr.io/div/dgs:prod-7f966bc-20220531094748 #{"$imagepolicy": "flux-system:div-dgs"}
       environment:
         VAR_BASE: tr-1
+        CTSC_OPENING_HOURS: "8am to 6pm, Monday to Friday"
```

Comment:
Shouldn't the hours be like this: "Monday - Friday - Phones and Webchat are open 8am-6pm." as this would supersede the work for DFR-1092? See: https://tools.hmcts.net/jira/browse/DFR-1092